edi_voxel/models/uom_uom.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class UoM(models.Model):
+    _inherit = "uom.uom"
+
+    voxel_code = fields.Selection(
+        selection=[
+            ("Unidades", "Unidades"),
+            ("Kgs", "Kgs"),
+            ("Lts", "Lts"),
+            ("Lbs", "Lbs"),
+            ("Cajas", "Cajas"),
+            ("Bultos", "Bultos"),
+            ("Palets", "Palets"),
+            ("Horas", "Horas"),
+            ("Metros", "Metros"),
+            ("MetrosCuadrados", "MetrosCuadrados"),
+            ("Contenedores", "Contenedores"),
+            ("Otros", "Otros"),
```

Comment:
Do you think to change and define in english?

Reply:
I think not. Look at the conversation between @pedrobaeza and @bealdav above

Reply:
Comment about it because other models contains some info in English but OK